### ISSUE TITLE
Automated related links not being generated

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_services#72]
Assume this is due to a rewrite of how this works.
Currently the method `buildAutomated` in `modules/localgov_services_page/src/Plugin/Block/ServicesRelatedLinksBlock.php` returns an empty array.
````php 
  /**
   * Automatically builds a list of links based on the most relevant pages.
   *
   * @todo Decide how the automated link generation should work.
   *
   * @return array
   *   Array of links.
   */
  private function buildAutomated() {
    // Return an empty array for the time being.
    return [];
````

**Comment by andybroomfield** [comment]
Flagging this up as this will be an issue for BHCC as we do use this functionality. 
Should we bring this up at next product groups @willguv @danchamp about possible inclusion.

**Comment by willguv** [comment]
@andybroomfield thanks for flagging this. Is there a bug or did this functionality not make it across during Alpha?

**Comment by andybroomfield** [comment]
@willguv No, it returning early and commenting out. I'll take a look and see if restoring this feature is feasable, although I never fully understood the algorithm its meant to use, is it just pages with the most common tags?

**Comment by finnlewis** [comment]
@ekes recalls that the product group were going to discuss how this should work.

Complexities around: 
 - how people expect this to work
 -  how the ordering should work

Currently there is a checkbox that does not do anything. 

@msayoung suggested that we could do some research into how content designers want this to work. 

This is an overarching information architecure question.

**Comment by finnlewis** [comment]
But! we should fix the checkbox!

**Comment by finnlewis** [comment]
https://www.rbkc.gov.uk/rbkc-digital-services/how-website-structured

**Comment by ekes** [comment]
> But! we should fix the checkbox!

I don't see a checkbox any more?

But something should be logged about Services Navigation - especially the sublanding page and use of topics.
A 'Product group decision' tag?

**Comment by andybroomfield** [comment]
<img width="859" alt="Screenshot 2021-10-12 at 2 38 54 pm" src="https://user-images.githubusercontent.com/1467480/136966846-45e59bac-ccd6-4217-93d0-4318a7fcdb3e.png">
Its 

> Replace automatically generated links

Which won't do anything if left unchecked (you have to replace the links).

**Comment by andybroomfield** [comment]
As this code has never been supported in LocalgovDrupal, best to now remove it and the checkbox which doesn't do anything.

**Comment by andybroomfield** [comment]
To flag this as a releated issue of #270 as when testing, if the checkbox isn't checked then the releated links won't event show.
